### ISSUE TITLE
Breakdown Animal Record Form Template

### ISSUE THREAD

**Issue by SanaKhan1997** [holtzmak/Humane-Transport#29]
EDIT: Issue split from #24, see #24 for further information
- Add required information for animal record
- There is no specific format requirements for this form
- Are there any specific rules & regulations targeting animal record

**Comment by holtzmak** [comment]
Referring to Appendix 4 for Animal Record Template on the [CFIA website](https://www.inspection.gc.ca/animal-health/humane-transport/health-of-animals-regulations-part-xii/eng/1582126008181/1582126616914#app4)

**Comment by holtzmak** [comment]
@SanaKhan1997 [Associated Wiki page](https://github.com/holtzmak/Humane-Transport/wiki/Animal-Record-Transport-Template-Breakdown) ready for review

**Comment by SanaKhan1997** [comment]
Thanks this looks great! Just noticed few questions that I think should be covered:

1. Is there any specified time period that the record needs to be retrievable. For example, transfer of care needs stored for minimum 2 years?
2. Does consignee have the right to refuse shipment if they notice inconsistencies or comprised animal welfare conditions? If yes are they required to document their decision on animal record or inform CFIA?

Also could you mention the HAR section(probably 154.1) that covers animal record for people who would want to check the rules & regulation from government site reference.

**Comment by holtzmak** [comment]
I've added relevant HAR Pt. XII sections where I felt necessary, and made note of the section of the Interpretive Guidance document as pertaining to compromised animals.

tldr; The record should be kept for 2 years (looks like most records should be kept 2 years) and the consignee is not allowed to refuse a shipment of animals if they are compromised - consignees must immediately provide care for compromised animals and complain to the CFIA about it.

**Comment by SanaKhan1997** [comment]
Awesome! this looks great and covers everything.
